go/vt/vtctl/vdiff_env_test.go

```diff
@@ -94,6 +80,17 @@ func newTestVDiffEnv(sourceShards, targetShards []string, query string, position
 	}
 	env.wr = wrangler.NewTestWrangler(env.cmdlog, env.topoServ, env.tmc)
 
+	dialerName := fmt.Sprintf("VDiffTest-%d", rand.Intn(1000000000))
```

Comment:
if we update this to take a `*testing.T` (or better, `testing.TB`), you can parameterize this to `fmt.Sprintf("VDiffTest-%s", t.Name())` and avoid any chance of collision entirely

Reply:
Good idea! I did that here (kept the rand int part for too): https://github.com/vitessio/vitess/pull/12546/commits/2d7e509e104c4e0a640a96d01ebf3398cf0133c5

Thanks!